instaclone/src/App.js

```diff
@@ -0,0 +1,38 @@
+import React, { Component } from 'react';
+import './App.css';
+import PostContainer from './components/PostContainer/PostContainer';
+import dummyData from './dummy-data';
+import SearchBar from './components/SearchBar/SearchBar';
+
+const appData = dummyData;
```

Comment:
appData is unnecessary because you could just assign your instagramData key to dummyData directly, cutting out that extra variable.